packages/teleport/src/Audit/EventList/EventList.tsx

```diff
@@ -27,7 +27,14 @@ import EventDialog from '../EventDialog';
 import { displayDateTime } from 'shared/services/loc';
 
 export default function EventList(props: Props) {
-  const { clusterId, events = [], search = '', fetchMore, fetchStatus } = props;
+  const {
+    clusterId,
+    events = [],
+    search = '',
+    fetchMore,
+    fetchStatus,
+    pageSize = 100,
```

Comment:
lets default to 50, like we had before
```suggestion
    pageSize = 50,
```